packages/cli/rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import esbuild from 'rollup-plugin-esbuild'
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import pkg from './package.json'
+
+const entries = [
+  'src/index.ts'
+]
+
+const external = [
+  ...Object.keys(pkg.dependencies),
+  ...Object.keys(pkg.peerDependencies),
+]
+
+export default ({ watch }) => [
+  {
+      input: entries,
```

Comment:
These are all the files we want to **process**, the only one in our case is `src/index.ts`